<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorClass</class>
 <widget class="QMainWindow" name="EditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LBANet Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>LBA-net.ico</normaloff>LBA-net.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>217</green>
         <blue>218</blue>
        </color>
       </brush>
      </property>
      <widget class="QWidget" name="worldsubwindow">
       <property name="windowTitle">
        <string>Subwindow</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_World"/>
    <addaction name="actionLoad_World"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_World">
   <property name="text">
    <string>New World</string>
   </property>
  </action>
  <action name="actionLoad_World">
   <property name="text">
    <string>Open World</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections/>
</ui>
